examples/experimental/usernet.yaml

```diff
@@ -0,0 +1,15 @@
+# Example to run vz instance with lima usernet enabled
```

Comment:
> `usernet.yaml`

`usernet-v2.yaml`, or maybe `net-user-v2.yaml`

Reply:
Will rename this once the name is decided in above comment

Reply:
Done